### PR TITLE
#783 fix - Tootip.left should be >=0

### PR DESCRIPTION
**Pull request by danikf**
Hi, I am not sure if other positions could be fixed this way without contradictions, but this one seams to be logical ...

### CODE REVIEW

File: src/MatBlazor/Components/MatTable/MatTable.razor [hunk 1]
```diff
@@ -269,18 +270,20 @@
...
-                .GetType()
-                .GetProperty(PagingDataPropertyName)
-                .GetValue(PagedData);
-        var Count = Convert.ToInt32(
-            PagedData
-                .GetType()
-                .GetProperty(PagingRecordsCountPropertyName)
-                .GetValue(PagedData)
-            );
+        var PagedData = await Http.GetStringAsync(ApiUrl + SearchTermParam(SearchTerm));
```
Comment (Christian-Oleson) [inline src/MatBlazor/Components/MatTable/MatTable.razor:273]: PagedData is not like the rest of the variables. Let's go to pageData

File: src/MatBlazor/Components/MatTable/MatTable.razor [hunk 2]
```diff
@@ -269,18 +270,20 @@
...
+        
+        var opt = new JsonSerializerOptions()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        };
+        
+        var items = JsonSerializer.Deserialize<List<TableItem>>(itemsString, opt);
+        
+        Items = items;
+        var Count = root.GetProperty(PagingRecordsCountPropertyName).GetInt32();
```
Comment (Christian-Oleson) [inline src/MatBlazor/Components/MatTable/MatTable.razor:286]: same issue with count

File: src/MatBlazor/Components/MatTable/MatTable.razor [hunk 3]
```diff
@@ -269,18 +270,20 @@
...
+        var itemsString = root.GetProperty(PagingDataPropertyName).GetRawText();
+        
+        var opt = new JsonSerializerOptions()
+        {
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+        };
+        
+        var items = JsonSerializer.Deserialize<List<TableItem>>(itemsString, opt);
+        
+        Items = items;
```
Comment (Christian-Oleson) [inline src/MatBlazor/Components/MatTable/MatTable.razor:285]: why not set Items directly?

**Review by Christian-Oleson — CHANGES_REQUESTED** [review]

File: src/MatBlazor/Components/MatTable/MatTable.razor (hunk 3 above)
Comment (danikf) [reply src/MatBlazor/Components/MatTable/MatTable.razor:285]: Hi,
it seams that "change base branch" operations took additional changes from different PR into PR. I am not the author of these changes (may be PR #769). I am not sure if original author will be notified ...

**Comment by danikf** [comment]
Hi,
it seams that "change base branch" operations took additional changes from different PR into PR. I am not the author of these changes (may be PR #769). I am not sure if original author will be notified ...
D

**Comment by SamProf** [comment]
Thank you for your contribution 👍!

**Pull request merged**